pyibisami/ami_model.py

```diff
@@ -304,13 +304,13 @@ def initialize(self, init_object):
                 self._num_aggressors,
                 self._sample_interval,
                 self._bit_time,
-                "{}".format(self._ami_params_in),  # Prevents model from mucking up our input parameter string.
```

Comment:
I notice, now that I'm looking at the final submitted review, that this change is marked "Outdated" by Git. Does that mean that I commented, before seeing _all_ the changes you made to this area of the code?

(I was going commit-by-commit, before I realized that I could view all commits at once. Sorry, I'm a bit of a novice at using Git pull request and code review tools.)

Reply:
That was a miss. I will revert this.  empy doesn't give any debug context whatsoever and it took awhile to find why it was segment faulting when opening the`.so`

Reply:
Actually, this was part of the seg fault.  I have to utf encode what I get back from the cdll since strings are completely different in python3.  So in my last commit, I had to take the format off because it was striping the encoding that I did a few lines before.
`self._ami_params_in = ami_params_in.encode("utf-8")`